5-infrastructure/modules/base-env/README.md

```diff
@@ -0,0 +1,75 @@
+# Base Env Module
```

Comment:
This is in the other PR, lets remove this from here and we can rebase when the other PR is merged